crates/iceberg/src/metadata_scan.rs

```diff
@@ -128,6 +137,135 @@ impl<'a> SnapshotsTable<'a> {
     }
 }
 
+/// Manifests table.
+pub struct ManifestsTable<'a> {
+    metadata_table: &'a MetadataTable,
+}
+
+impl<'a> ManifestsTable<'a> {
+    fn partition_summary_fields(&self) -> Vec<Field> {
+        vec![
+            Field::new("contains_null", DataType::Boolean, false),
+            Field::new("contains_nan", DataType::Boolean, true),
+            Field::new("lower_bound", DataType::Utf8, true),
+            Field::new("upper_bound", DataType::Utf8, true),
+        ]
+    }
+
+    fn schema(&self) -> Schema {
```

Comment:
We might want to make this `pub`, so engines can get the schema first without fetching the data.

Reply:
fixed in 83e8811